### ISSUE TITLE
Generation of result-based remediation roles does not work for tailored profiles.

### ISSUE THREAD

**Issue by matejak** [OpenSCAP/scap-workbench#148]
Scap-workbench won't support generation of result-based remediation roles from tailored profiles even after #144 is merged. This is caused by a deficiency in openscap (https://github.com/OpenSCAP/openscap/issues/902) that however may be fixed on the workbench side.
The fix is not trivial - requires access to the scanning session, which is the property of `MainWindow`. The remediation generation is triggered from the `ResultsViewer` window.
Fixing this would require decoupling UI from the scanning according to the MVC pattern, which is out of scope for the mentioned PR #144 .

**Comment by cipherboy** [comment]
@matejak Since https://github.com/OpenSCAP/openscap/issues/902 and https://github.com/OpenSCAP/openscap/pull/1253 were merged in 1.2.18 (and presumably, also in `maint-1.3`?) -- are we fine to close this?
